app/assets/stylesheets/thyme.scss

```diff
@@ -341,3 +418,234 @@ figure {
  {
    background-color: black;
  }
+
+
+
+/* annotation modal */
+#annotation-modal {
+  .modal-content {
+    width: 100%;
+  }
+  #annotation-modal-content {
+    width: 100%;
+  }
+  #preview-toggle {
+    input:checked + .slider {
+      background-color: #0D6EFD;
+    }
+    .switch {
+      vertical-align: middle;
+    }
+  }
+  #annotation-modal-preview {
+    height: 543px;
+    width: 35%;
+    word-wrap: break-word;
+    overflow-y: scroll;
+  }
+  #annotation_comment {
+    height: 150px;
+    resize: none;
+  }
+  #annotation_category_text {
+    width: 200px;
+  }
+  input[type=checkbox] {
+    width: 18px;
+    height: 18px;
+  }
+}
+
+#emergency-link {
+  text-align: center;
+}
+
+#markers {
+  position: relative;
+  top: -2px;
+  width: 0;
+  height: 0;
+  display: flex;
+  cursor: pointer;
+}
+
+#annotation-caption {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+#annotation-infobar {
+  height: 2em;
+  color: black;
+  overflow-y: scroll;
+  padding: 3px;
+  border-left: 1px solid darkgray;
+  border-right: 1px solid darkgray;
+  border-top: 1px solid darkgray;
+}
+
+#annotation-comment {
+  margin-left: 3px;
+  margin-right: 3px;
+  overflow-y: scroll;
+  height: 85%;
+}
+
+#annotation-buttons {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 10%;
+  margin-right: 10%;
+  margin-bottom: 15px;
+  margin-top: auto;
+}
+
+
+/* A collection of all colors.
+Be aware that color changes must also be written in the
+AnnotationHelper! */
```

Comment:
No, please avoid these things where we have duplicated code and then even a comment telling: when you touch this, you also have to touch this. See my proposal in the `_form.html.erb` file. Short form: keep it here, remove it from the helper.